Google-Summer-of-Code-2024.md

```diff
@@ -246,11 +246,13 @@ Here's some advice about proposals and milestone timeline planning that we colla
 
 ## Selection Criteria
 
-In order to select contributors for GSoC, we will mainly be looking at three things (the first of this is the most important, and the other two are of roughly equal weight):
+To select contributors for GSoC, we will evaluate candidates based on a set of criteria designed to ensure we select individuals who not only possess the necessary skills but also demonstrate the ability to contribute effectively to the project. The criteria are as follows, listed in order of significance::
 
-- Whether the contributor has the required skills for the project
-- The quality of the submitted proposal
-- Our prior experience working with the contributor (do they keep commitments, communicate well, demonstrate independence/initiative/responsiveness, help others, etc.)
+- **Primary Criterion: Required Skills for the Project** - This is the most critical factor in our selection process. A contributor must have the necessary skills for the project. Lack of these skills is a deal-breaker and can lead to immediate rejection of the proposal.
+
+- **Secondary Criteria** (of Equal Importance):
```

Comment:
I think you can change "Equal Importance" to "equal importance"

Reply:
Done!